prisma/schema.prisma

```diff
@@ -0,0 +1,50 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+model User {
+  id                 Int           @id @default(autoincrement())
+  firebaseUid        String?
+  name               String
+  email              String        @unique
+  digitaloceanApiKey String?
+  minecraftUsername  String?
+  worlds             WorldAccess[]
```

Comment:
Is the idea here that we can access which Worlds a given User is part of?

Reply:
Yep, the WorldAccess table is a pivot table, defining the relationship between User and World. So for the models, we can get its relations. Like `user.worlds` or `world.users`. It uses this table and relation to define those.